src/plugins/controls/public/actions/delete_control_action.tsx

```diff
@@ -94,12 +99,23 @@ export class DeleteControlAction implements Action<EmbeddableApiContext> {
   public async execute({ embeddable }: EmbeddableApiContext) {
     if (!isApiCompatible(embeddable)) throw new IncompatibleActionError();
 
-    this.openConfirm(ControlGroupStrings.management.deleteControls.getSubtitle(), {
-      confirmButtonText: ControlGroupStrings.management.deleteControls.getConfirm(),
-      cancelButtonText: ControlGroupStrings.management.deleteControls.getCancel(),
-      title: ControlGroupStrings.management.deleteControls.getDeleteTitle(),
-      buttonColor: 'danger',
-    }).then((confirmed) => {
+    this.openConfirm(
+      i18n.translate('controls.controlGroup.management.delete.sub', {
+        defaultMessage: 'Controls are not recoverable once removed.',
+      }),
+      {
+        confirmButtonText: i18n.translate('controls.controlGroup.management.delete.confirm', {
+          defaultMessage: 'Delete',
+        }),
+        cancelButtonText: i18n.translate('controls.controlGroup.management.delete.cancel', {
+          defaultMessage: 'Cancel',
+        }),
+        title: i18n.translate('controls.controlGroup.management.delete.deleteTitle', {
+          defaultMessage: 'Delete control?',
+        }),
```

Comment:
Moved to inline `i18n` string rather than having the `ControlGroupStrings` - this is better for bundle size, since that file includes a lot of translations that are **not** used here.